### ISSUE TITLE
[REVIEW] EddieHub Python API

### ISSUE THREAD

**Issue by FayasNoushad** [EddieHubCommunity/support#2408]
### Project link

https://github.com/FayasNoushad/EddieHub

**Comment by Panquesito7** [comment]
A few suggestions:

- The contributing guidelines should be updated with proper guidelines of how to contribute.
- Missing code of conduct.
- Missing issue and pull request templates.
- Maybe change the repository description to `A Python package for the EddieHub API`.
- Add a security policy so people can know how to securely report bugs/vulnerabilities.

Overall, everything else looks great! Good job! 😄👍

**Comment by FayasNoushad** [comment]
> A few suggestions:
> 
> * The contributing guidelines should be updated with proper guidelines of how to contribute.
> * Missing code of conduct.
> * Missing issue and pull request templates.
> * Maybe change the repository description to `A Python package for the EddieHub API`.
> * Add a security policy so people can know how to securely report bugs/vulnerabilities.
> 
> Overall, everything else looks great! Good job!

If you interested in contributing you can contribute.

**Comment by schmelto** [comment]
Consider adding an example response of the api in the "usage"

**Comment by FayasNoushad** [comment]
> Consider adding an example response of the api in the "usage"

Ok, I will add.

**Comment by FayasNoushad** [comment]
> Consider adding an example response of the api in the "usage"

[![Screenshot](https://user-images.githubusercontent.com/76828314/133060496-8f6d166c-b1d1-4ef9-acf8-837ef3ad0b0d.png)](https://github.com/FayasNoushad/EddieHub/#Usage)

Check this.

**Comment by schmelto** [comment]
> > Consider adding an example response of the api in the "usage"
> 
> [![Screenshot](https://user-images.githubusercontent.com/76828314/133060496-8f6d166c-b1d1-4ef9-acf8-837ef3ad0b0d.png)](https://github.com/FayasNoushad/EddieHub/#Usage)
> 
> Check this.
> 

Yes but from there the result of the print statement or something like that

**Comment by FayasNoushad** [comment]
> > > Consider adding an example response of the api in the "usage"
> > 
> > 
> > [![Screenshot](https://user-images.githubusercontent.com/76828314/133060496-8f6d166c-b1d1-4ef9-acf8-837ef3ad0b0d.png)](https://github.com/FayasNoushad/EddieHub/#Usage)
> > Check this.
> 
> Yes but from there the result of the print statement or something like that

Ok
